tests/Unit/Test_ActionTesting.cpp

```diff
@@ -181,6 +181,27 @@ struct SimpleActionMockMetavariables {
   enum class Phase { Initialization, Testing, Exit };
 };
 
+struct MockMetavariablesWithGlobalCacheTags {
+  using component_list = tmpl::list<
+      component_for_simple_action_mock<SimpleActionMockMetavariables>>;
+  using const_global_cache_tag_list = tmpl::list<ValueTag,PassedToB>;
+
+  enum class Phase { Initialization, Testing, Exit };
+};
+
+void test_mock_runtime_system_constructors() {
+  using metavars = MockMetavariablesWithGlobalCacheTags;
+  // Test whether we can construct with tagged tuples in different orders.
+  ActionTesting::MockRuntimeSystem<metavars> runner1{
+      tuples::TaggedTuple<ValueTag, PassedToB>{3, 7}};
+  ActionTesting::MockRuntimeSystem<metavars> runner2{
+      tuples::TaggedTuple<PassedToB, ValueTag>{7, 3}};
+  CHECK(Parallel::get<ValueTag>(runner1.cache()) ==
```

Comment:
could you also check that `runner1` has the correct values in the cache? If `runner1` and `runner2` were both default constructed this test would still pass